i18n/ads-language-pack-de/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 All notable changes to the "ads-language-pack-de" language pack will be documented in this file.
 
 ## [Released]
+* December 1, 2022 - Release for Azure Data Studio 1.41
```

Comment:
This definitely isn't going to be accurate, so maybe wait to update the changelogs until we actually start doing the release.